ios/sentry.properties

```diff
@@ -1,4 +1,4 @@
-defaults.url=https://sentry.io
+defaults.url=https://sentry.io/
 defaults.org=zulip
 defaults.project=zulip-mobile
-auth.token=ADD-DSN-HERE
+auth.token=ADD-TOKEN-HERE
```

Comment:
As specified https://github.com/getsentry/react-native-sentry/issues/140#issuecomment-318572188 here its auth token there and not dsn!